### PR TITLE
[ACM-11206] Add ConfigMap for cross product search config

### PR DESCRIPTION
**Pull request by zlayne**
# Description

Adds a ConfigMap that will house the suggested searches so users can customize searches and/or remove the default searches to increase search page performance. 
Additionally allows for other data fields to be included in the future for other pages that use search. 

## Related Issue

https://issues.redhat.com/browse/ACM-11206

## Changes Made

Only change is the addition of the new ConfigMap `console-search-config`

## Screenshots (if applicable)

Add screenshots or GIFs that demonstrate the changes visually, if relevant.

## Checklist

- [ ] I have tested the changes locally and they are functioning as expected.
- [ ] I have updated the documentation (if necessary) to reflect the changes.
- [ ] I have added/updated relevant unit tests (if applicable).
- [ ] I have ensured that my code follows the project's coding standards.
- [ ] I have checked for any potential security issues and addressed them.
- [ ] I have added necessary comments to the code, especially in complex or unclear sections.
- [ ] I have rebased my branch on top of the latest main/master branch.

## Additional Notes

Add any additional notes, context, or information that might be helpful for reviewers.

## Reviewers

Tag the appropriate reviewers who should review this pull request. To add reviewers, please add the following line: `/cc @reviewer1 @reviewer2`

## Definition of Done

- [ ] Code is reviewed.
- [ ] Code is tested.
- [ ] Documentation is updated.
- [ ] All checks and tests pass.
- [ ] Approved by at least one reviewer.
- [ ] Merged into the main/master branch.

### CODE REVIEW

**Comment by zlayne** [comment]
FYI @jlpadilla

File: pkg/templates/charts/toggle/console/templates/console-search-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: console-search-config
+  labels:
+    app: console
+    component: 'console'
+    chart: console-chart-{{ .Values.hubconfig.hubVersion }}
+    release: console-chart
+data:
```
Comment (jlpadilla) [inline pkg/templates/charts/toggle/console/templates/console-search-configmap.yaml:13]: @zlayne FYI, you could make the string look like json with the `|` character. Here's an example.
https://stackoverflow.com/a/61654051

Comment (zlayne) [reply pkg/templates/charts/toggle/console/templates/console-search-configmap.yaml:13]: This looks much better I'll change that

**Pull request merged**